{"timestamp":"2025-09-29T16:04:07.030831","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-09-29T16:04:07.033566","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/etl_dags.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-09-29T16:04:09.794850Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-29T16:04:09.796451Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-29T16:04:09.797498Z","level":"info","event":"Current task name:extracting","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-29T16:04:09.798185Z","level":"info","event":"Dag name:etl_dag","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-29T16:04:09.798996","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"TypeError","exc_value":"download_from_s3() missing 3 required positional arguments: 'bucket', 'object_name', and 'local_file_path'","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":920,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1215,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":216,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":239,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-09-29T16:04:09.870594Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-29T16:04:09.871795Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-29T16:04:09.872777Z","level":"info","event":"Task:<Task(PythonOperator): extracting>","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-29T16:04:09.873610Z","level":"info","event":"Failure caused by download_from_s3() missing 3 required positional arguments: 'bucket', 'object_name', and 'local_file_path'","chan":"stdout","logger":"task"}
